fix broken links to Project Week youtube videos

jessicabartley opened the pull request.

jessicabartley committed:
fixed broken links

jessicabartley committed:
Merge branch 'project-page' of https://github.com/ABCD-ReproNim/materials into project-page

jessicabartley commented:
@tsalo not sure how to made the video links I added show up as embedded/as they do in the template pages. Do you know how?

tsalo commented:
The current change will just hyperlink to the video. If you want to actually embed the video there, I think you can use the same HTML we're using elsewhere:

https://github.com/ABCD-ReproNim/materials/blob/c16bb4830e9af106d85608baaba151e92957a2f9/_layouts/post.html#L84-L88

jessicabartley committed:
instantiate 2022 material pages

instantiate 2022 material pages as copies of existing material pages. Edits needed include date changes and text updates.

jessicabartley committed:
Remove last edits - wrong branch

Previous edits were made on the incorrect branch. Removing them.

tsalo commented:
Can this be closed?
